app/helpers/react_components/student/request_mentoring_button.rb

```diff
@@ -0,0 +1,30 @@
+module ReactComponents
+  module Student
+    class RequestMentoringButton < ReactComponent
+      def to_s
+        super(
+          "student-request-mentoring-button",
+          {
+            request: request,
+            links: {
+              mentor_request: Exercism::Routes.new_track_exercise_mentor_request_path("$TRACK_SLUG", "$EXERCISE_SLUG")
```

Comment:
Nice. You're using the `$TRACK_SLUG` and `$EXERCISE_SLUG` arguments as placeholders in the URL :)